src/storage/txn/scheduler.rs

```diff
@@ -1553,14 +1572,23 @@ impl Scheduler {
         }
     }
 
-    fn shutdown(&mut self) -> Result<()> {
+    fn on_tick(&mut self) {
+        // TODO
+    }
+
+    fn shutdown(&mut self) {
         if let Err(e) = self.worker_pool.stop() {
-            return Err(Error::Other(box_err!("{:?}", e)));
+            panic!("scheduler run err:{:?}", e);
```

Comment:
log an error here.

Reply:
don't panic

Reply:
in previous code, when err is returned, it will panic, maybe we should keep it that way?